commercetools-convenience/src/test/java/io/sphere/sdk/queries/QueryExecutionUtilsTest.java

```diff
@@ -51,9 +51,24 @@ public void onUnsortedResponses() throws Exception {
         withClient(clientWithDelays(16), list -> assertThat(list).isSortedAccordingTo(categoryComparator).hasSize(16));
     }
 
+    @Test
+    public void onGetTotalNumberOfPages() {
```

Comment:
Great that you added a test for this method 👍

Reply:
corner cases are missing, like:
  - [ ] `totalElements = 0`
  - [ ] `totalElements == pageSize`
    - [ ] `totalElements == pageSize - 1`
    - [ ] `totalElements == pageSize + 1`
  - [ ] `pageSize == 1`

Reply:
63b893d